knowledge_base/Environment-Variables.md

```diff
@@ -221,12 +229,17 @@ If set to `e2e-test`, the app will bypass usage of real ETH RPCs and use the moc
 
 Owner: Ian Rowan
 
+## EVM_CE_POLL_INTERVAL
+
+Required for testing contests. Number of milliseconds. `10000` is recommended; if the number is too low, millions of requests will be made.
+
 ## FALLBACK_NODE_DURATION_S
 
-Optional. Defaults to 5 minutes (300 seconds).
-This is number, in seconds. It configures the length of time we will use a community-maintained public endpoint if a given ChainNode fails.
-After this time, the server will try the original DB endpoint again.
+Optional. Defaults to 5 minutes (300 seconds). This is number, in seconds. It configures the length of time we will use a community-maintained public endpoint if a given ChainNode fails. After this time, the server will try the original DB endpoint again.
 
+## FLAG_CONTEST
+
+Boolean toggle to enable contests. Required for testing contests.
```

Comment:
I think this is not an env var, but a feature flag enabled via Unleashed. Need to confirm with @kurtisassad or @masvelio

Reply:
Indeed this is not an environment variable and is no longer set in Heroku config variables (only Unleash).